nautobot/docs/installation/pip-install.md

```diff
@@ -0,0 +1,234 @@
+
+Optional dependencies can be installed by running `pip3 install -r /opt/nautobot/local_requirements.txt` while activated in the virtualenv. If you decide to use any [Nautobot plugins](../../plugins), they should be listed in this file.
+
+### Configuring NAPALM
+
+Nautobot provides built-in support for the [NAPALM automation](https://napalm-automation.net/) library, which allows Nautobot to fetch live data from devices and return it to a requester via its REST API. The [`NAPALM_USERNAME`](../../configuration/optional-settings#napalm_username) and [`NAPALM_PASSWORD`](../../configuration/optional-settings#napalm_password) configuration parameters define the credentials to be used when connecting to a device.
+
+To use NAPALM, add `napalm` to your `local_requirements.txt` so that it can be installed and kept up to date:
+
+```no-highlight
+(nautobot) $ echo napalm >> /opt/nautobot/local_requirements.txt
+```
+
+### Remote File Storage
+
+By default, Nautobot will use the local filesystem to store uploaded files. To use a remote filesystem, install the [`django-storages`](https://django-storages.readthedocs.io/en/stable/) library and configure your [desired storage backend](/configuration/optional-settings/#storage_backend) in `nautobot_config.py`.
+
+To use remote file storage, add `django-storages` to your `local_requirements.txt` so that it can be installed and kept up to date:
+
+```no-highlight
+(nautobot) $ echo django-storages >> /opt/nautobot/local_requirements.txt
+```
+
+## Prepare the Database
+
+Before Nautobot can run, the database migrations must be performed to prepare the database for use. This will populate the database tables and relationships:
+
+```no-highlight
+(nautobot) $ nautobot-server migrate
+```
+
+## Create a Superuser
+
+Nautobot does not come with any predefined user accounts. You'll need to create a super user (administrative account) to be able to log into Nautobot. Specifying an email address for the user is not required, but be sure to use a very strong password.
+
+```no-highlight
+(nautobot) $ nautobot-server createsuperuser
+```
+
+## Create Static Directories
+
+Nautobot relies upon many static files including:
+
+- `git` - For storing [Git repositories](../../models/extras/gitrepository)
+- `jobs` - For storing [custom Jobs](../../additional-features/jobs)
+- `media` - For storing [uploaded images and attachments](../../configuration/optional-settings/#media_root) (such as device type images)
+- `static` - The home for [CSS, JavaScript, and images](../../configuration/optional-settings/#static_root) used to serve the web interface
+
+Each of these have their own corresponding setting that defined in `nautobot_config.py`, but by default they will all be placed in `NAUTOBOT_ROOT` unless you tell Nautobot otherwise by customizing their unique variable.
```

Comment:
```suggestion
Each of these can be individually placed by setting variables in `nautobot_config.py`, but by default they will all be placed in `NAUTOBOT_ROOT` unless you tell Nautobot otherwise by customizing their unique variable.
```